ui/lib/config-ui/addon/components/login-settings/page/details.hbs

```diff
@@ -0,0 +1,67 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+{{#if @rule}}
+  <PageHeader as |p|>
+    <p.top>
+      <Page::Breadcrumbs @breadcrumbs={{@breadcrumbs}} />
+    </p.top>
+    <p.levelLeft>
+      <h1 class="title is-3">
+        {{@rule.name}}
+      </h1>
+    </p.levelLeft>
+  </PageHeader>
+
+  <nav class="tabs" aria-label="navigation for rule details">
+    <ul>
+      <li>
+        <LinkTo @route="login-settings.rule.details" @model={{@rule}} data-test-rule-tab>
+          Details
+        </LinkTo>
+      </li>
+    </ul>
+  </nav>
+
+  <Toolbar>
+    <ToolbarActions>
+      {{#if (has-capability this.capabilities "delete" id=@rule.name)}}
+        <Hds::Button
+          @text="Delete Rule"
+          @color="secondary"
+          class="toolbar-button"
+          {{on "click" (fn (mut this.showConfirmModal) true)}}
+          data-test-rule-delete
+        />
+      {{/if}}
+    </ToolbarActions>
+  </Toolbar>
+  {{#each-in @rule as |key value|}}
+    {{#if (eq key "default_auth_type")}}
+      <InfoTableRow @alwaysRender={{true}} @label="Default method" @value={{value}} />
+    {{else if (eq key "backup_auth_types")}}
+      <InfoTableRow @alwaysRender={{true}} @label="Backup methods" @value={{value}} />
+    {{else if (eq key "disable_inheritance")}}
+      <InfoTableRow @alwaysRender={{true}} @defaultShown={{true}} @label="Inheritance" @value={{stringify (not value)}} />
+    {{else}}
+      <InfoTableRow @alwaysRender={{true}} @label={{capitalize key}} @value={{value}} />
+    {{/if}}
+  {{/each-in}}
+
+{{else}}
+  {{#if @error}}
+    <Page::Error @error={{@error}} />
+  {{else}}
+    <EmptyState @title="No rule found" @message="The rule you are looking for does not exist." data-test-empty-state />
```

Comment:
For test selectors on Empty State, we use [these 4](https://github.com/hashicorp/vault/blob/main/ui/tests/helpers/general-selectors.ts#L32-L35). These selectors are on the `EmptyState` component. However, if you want to select the EmptyState component itself to see if it exists, you can use something like `data-test-empty-state="config-details"`. My recommendation, is when you're writing your tests, you check the title and message and not just that the empty state exists.